docs/troubleshooting/report-scylla-problem.rst

```diff
@@ -16,40 +16,59 @@ Make sure the Scylla system logs are configured properly to report info level me
 .. note:: 
    If you are unsure which reports need to be included, `Open a support ticket or GitHub issue`_ and consult with the ScyllaDB team. 
 
-Health Check Report
-^^^^^^^^^^^^^^^^^^^
 
-The Health Check Report is a script which generates:
+.. _report-scylla-problem-scylla-doctor:
 
-* An archive file (output_files.tgz) containing configuration data (hardware, OS, Scylla SW, etc.) 
-* System logs 
-* A Report file ``(<node_IP>-health-check-report.txt)`` based on the collected info.
+Scylla Doctor Vitals
+^^^^^^^^^^^^^^^^^^^^^
 
-If your node configuration is identical across the cluster, you only need to run the script once on one node. If not you will need to run the script on multiple nodes.
+Scylla Doctor is a troubleshooting tool that checks the node status regarding 
+system requirements, configuration, and tuning. The collected information is 
+output as a ``.vitals.json`` file. You need to run the tool **on every node 
+in the cluster**. 
 
-.. note:: In order to generate a full health check report, scylla-server and scylla-jmx must be running. Note that the script will alert you if either one is not running. By default, the script looks for scylla-jmx via port 7199. If you are running it on a different port, you will have to provide the port number at runtime. 
+#. Download the Scylla Doctor executable ``scylla_doctor.pyz`` from 
+   https://downloads.scylladb.com/downloads/scylla-doctor/latest/. 
+#. Copy ``scylla_doctor.pyz`` file to all nodes in the cluster.
+#. Execute the following command from the directory where you copied 
+   ``scylla_doctor.pyz`` on each node in your cluster:
 
-Prepare Health Check Report
-...........................
+   .. code:: shell
 
-**Procedure:**
+      sudo ./scylla_doctor.pyz --save-vitals <unique-host-id>.vitals.json
+ 
+   Make sure you provide a unique host identifier in the filename, such as 
+   the host IP. Scylla Doctor will generate:
+   
+   * ``<unique-host-id>.vitals.json`` - Scylla Doctor vitals
+   * ``scylla_logs_<timestamp>.tar.gz`` - ScyllaDB logs
 
-1. Run the node healthcheck:
+   **Authenticated Clusters**
 
-.. code-block:: shell
+   If CQL authentication is enabled on the cluster, you need to additionally
+   provide CQL credentials with permissions to perform the ``DESCRIBE SCHEMA`` 
+   command using the following parameters:
 
-   node_health_check
- 
-The report generates output files. Once complete, a similar message is displayed:
+   .. code:: shell
 
-.. code-block:: shell
+      -sov CQL,user <CQL user name> -sov CQL,password <CQL password>
```

Comment:
Incorrect format: should be: `-sov CQL,user,<CQL user name> -sov CQL,password,<CQL password>`

The value of a `-sov` parameter is a coma separated triplet.